torchrl/objectives/returns/functional.py

```diff
@@ -88,3 +93,47 @@ def td_advantage_estimate(
     not_done = 1 - done.to(next_state_value.dtype)
     advantage = reward + gamma * not_done * next_state_value - state_value
     return advantage
+
+
+def td_lambda_advantage_estimate(
+    gamma: float,
+    lamda: float,
```

Comment:
lambda, missing the 'b' here

Reply:
I know but python doesn't like lambda :)

Reply:
maybe 'lambda_'?

Reply:
I've raised the issue with pytorch core and domain folks, i'll keep you posted
Seems like many ppl use `lmbda`

Reply:
ok, cool.